number_parser/parser.py

```diff
@@ -265,6 +265,7 @@ def parse_number(input_string, language=None):
     number_built = _build_number(normalized_tokens, lang_data)
     if len(number_built) == 1:
         return int(number_built[0])
+
```

Comment:
This is not what @noviluni meant, he meant after the last return. PEP8 asks that top-level function definitions are separated by 2 empty lines.

Reply:
Sorry for that misinterpretation, I have corrected it.